src/aat/resources/features/F-103/F-103_Test_Case_Access_Base_Dil_C2.td.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_guid_": "F-103_Test_Case_Access_Base_Dil_C2",
+	"method": "PUT",
+	"uri": "/cases/{cid}/users/{uid}",
+	"users": {
+    "invokingUser": {
+      "username": "befta.caseworker.caa.1@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userRichard": {
+      "username": "befta.caseworker.2.solicitor.2@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    },
+    "userDil": {
+      "username": "befta.caseworker.2.solicitor.3@gmail.com",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
```

Comment:
each account has it's respective password env var. please always use respective env vars.

Reply:
in general...

Reply:
Used user specific pwd, and added new entry for befta.caseworker.caa@gmail.com